src/fidesops/service/connectors/saas_connector.py

```diff
@@ -388,3 +376,35 @@ def mask_data(
 
     def close(self) -> None:
         """Not required for this type"""
+
+    @staticmethod
+    def _handle_errored_response(
+        saas_request: SaaSRequest, response: Response
+    ) -> Response:
```

Comment:
These could also be split out into a separate functions, instead of staticmethods, but I'm indifferent as we're probably only using these within the SaasConnector context.